dashboard/src/index.html

```diff
@@ -8,12 +8,15 @@
 
   <link href="img/favicon.ico" rel="shortcut icon" type="image/vnd.microsoft.icon" />
   <!-- STYLES -->
-  <link rel="stylesheet" type="text/css" href="css/index.css">
   <!-- build:css lib/css/main.min.css -->
   <link rel="stylesheet" type="text/css" href="components/bootstrap/dist/css/bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="components/font-awesome/css/font-awesome.min.css">
   <link rel="stylesheet" type="text/css" href="components/rdash-ui/dist/css/rdash.min.css">
   <!-- endbuild -->
+
```

Comment:
Hey guys, this is taking too long. I will try to be clear. 

The goal of using `Gulp` is to save both on the number of requests and the bytes send from the server.

The point of separating the `index.css` just to be easy to find out in which line is the CSS is counterproductive. 

That said, you can still make your own `Gulpfile.js` and opt not to minify and concatenate the CSS files. 

Thanks